CIP-0165/README.md

```diff
@@ -0,0 +1,417 @@
+---
+CIP: 165
+Title: Canonical Ledger State
+Category: Ledger
+Status: Proposed
+Authors:
+  - Nicholas Clarke <nicholas.clarke@moduscreate.com>
+  - Aleksandr Vershilov <alexander.vershilov@moduscreate.com>
+  - João Santos Reis <joao.reis@moduscreate.com>
+Implementors:
+  - Nicholas Clarke <nicholas.clarke@moduscreate.com>
+  - Aleksandr Vershilov <alexander.vershilov@moduscreate.com>
+  - João Santos Reis <joao.reis@moduscreate.com>
+Discussions:
+    - https://github.com/cardano-foundation/CIPs/pull/1083
+Created: 2025-08-18
+License: CC-BY-4.0
+---
+
+## Abstract
+
+This proposal defines the Simple Canonical Ledger State (SCLS), a stable, versioned, and verifiable file format for representing the Cardano ledger state. It specifies a segmented binary container with deterministic CBOR encodings, per-chunk commitments, and a manifest that enables identical snapshots across implementations, supports external tools (e.g., Mithril), and future-proofs distribution and verification of state.
+
+## Motivation: why is this CIP necessary?
+
+Ledger state serialisations are currently implementation details that may change over time. This makes them unsuitable as stable artifacts for distribution, signing, fast sync, or external tooling (e.g., db-sync, conformance testing, and Mithril checkpoints). Without a canonical format, two nodes at the same chain point can legitimately produce different byte streams for the same state, complicating verification and opening room for error in multi-implementation ecosystems.
+
+SCLS addresses these problems by:
+
+- specifying a canonical, language-agnostic container and encoding rules;
+- enabling streaming builds and partial verification (per-namespace roots);
```

Comment:
What sort of verification you have in mind? You mean data consistency rather than data validity, right?